code/modules/power/supermatter/supermatter.dm

```diff
@@ -2,6 +2,16 @@
 //Please do not bother them with bugs from this port, however, as it has been modified quite a bit.
 //Modifications include removing the world-ending full supermatter variation, and leaving only the shard.
 
+//Zap constants, speeds up targeting
+
+#define BIKE (6)
+#define COIL (BIKE - 1)
+#define ROD (COIL - 1)
+#define LIVING (ROD - 1)
+#define MACHINERY (LIVING - 1)
+#define OBJECT (MACHINERY - 1)
+#define LOWEST (OBJECT - 1)
```

Comment:
should this not go the otherway? otherwise this will start being negative at some point cause of someone not counting properly

Reply:
being negative doesn't actually effect the code, although going bottom up does seem a little more clean, so sure.